docs/api-docs/store-management/shipping/shipping-provider-api.mdx

```diff
@@ -37,13 +37,88 @@ BigCommerce makes a distinction between single-carrier and multi-carrier shippin
 
 ![Multi-carrier quote example](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Multi%20Carrier%20Example.png)
 
+### Carrier configurations
+
+The configurations schema defines the optional settings that can be applied to enhance the user experience.
+
+If you're unsure about the configurations you may need, you can leave them unspecified for now and request to add them later.
+
+There are two types of configuration options: "connection" and "settings". For each configuration option, please provide the following details:
+
+*Code*: An identifier for this option. This identifier and the merchant input value for this configuration will provided on the rate or the check and validate connection options provided endpoints.
+*Type*: Choose from valid types such as text, checkbox, select, multiselect, password.
+*Label*: Enter the text to be displayed on the BigCommerce control panel for this option.
+*Description*: Briefly describe the purpose of this option.
+*Required*: Specify whether this option is mandatory or not.
```

Comment:
The context in these descriptions seemed to be a little mixed.
`Choose from valid` vs `An identifier`
It would be good to have a consistent context.
My preference is more descriptive.
for instance `A simple boolean field which sets whether this option is mandatory`, `A brief description of this options purpose`